app/templates/profile.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "layout.html" %}
```

Comment:
Looked like we just had a little typo in here. I probably should have left this for a different branch, but this was pretty early in my journey. Wanted to confirm that my sign up / login was working.